disabledTestParser/issue_status.py

```diff
@@ -0,0 +1,37 @@
+import json
+import requests
+
+
+def json_parser(json_obj):
+    jsonLoad = json.loads(json_obj)
+    return jsonLoad["GIT_ISSUE"]
+
+def find_state(query_url):
+    params = {'accept': 'application/vnd.github.v3+json',
+              'state': 'all'}
+    response = requests.get(query_url, params=params).json()
+    json_str = json.dumps(response)
+    return json.loads(json_str)["state"]
```

Comment:
Could we add error handling here? i.e., what if the URL is invalid and we cannot get the state?

Reply:
I resolved it please review